src/App.js

```diff
@@ -37,10 +37,10 @@ function App() {
                 path="/:productDashboardID"
```

Comment:
This route is the same is landing page. I suggest you make it: "/<route>/:productDashboardID"